src/utilities/constants.hpp

```diff
@@ -0,0 +1,10 @@
+#ifndef ABLATELIBRARY_CONSTANTS_HPP
+#define ABLATELIBRARY_CONSTANTS_HPP
+
+#include <petscsystypes.h>
+
+/// Class Constants
```

Comment:
Can we wrap this in a class to prevent name conflicts?  Something like:

`namespace ablate::utilities {
class VectorUtilities {
   public:
     static const PetscReal sbc = 5.6696e-8;  //!< Stefan-Boltzman Constant (J/K)
 static const PetscReal pi = 3.1415926535897932384626433832795028841971693993;